src/jit/Backend.cpp

```diff
@@ -120,6 +127,9 @@ struct CompileContext {
         , trapLabel(nullptr)
         , memoryTrapLabel(nullptr)
     {
+        // Same calculation as in Instance::alingedSize() but that function is private.
+        sljit_sw offset = (sizeof(Instance) + sizeof(void*) - 1) & ~(sizeof(void*) - 1);
```

Comment:
Come to think of it, code duplication is not a good idea. Maybe we could make that function public.